inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -187,6 +191,9 @@ def is_record_relevant(obj, eng):
     if is_submission(obj, eng):
         return True
 
+    if _is_auto_approved(workflow_obj=obj):
+        return True
+
     if _is_auto_rejected(workflow_obj=obj):
         return False
```

Comment:
couldn't this whole function be simplified to `return is_submission(obj, eng) or _is_auto_approved(obj) or not _is_auto_rejected(obj)`?